dienynas.php

```diff
@@ -3,29 +3,30 @@
 error_reporting(E_ALL);
 $marksFilename = 'marks.txt';
 @$marksFile = fopen(@$marksFilename, "r");
-if (!file_exists($marksFilename) or !is_readable($marksFilename)) {
+if (!file_exists($marksFilename) or !is_writable($marksFilename)) {
     $errorMessage = "Nepavyksta atidaryti failo su mokinių pažymiais!";
     include 'errorTemplate.php';
     exit();
 }
 @$peopleFile = fopen(@$peopleFilename, "r");
-$peopleFilename = 'people.txt';
+$peopleFilename = 'students.csv';
 if (!file_exists($peopleFilename) or !is_readable($peopleFilename)) {
     $errorMessage = "Nepavyksta atidaryti failo su mokinių sąrašu!";
     include 'errorTemplate.php';
     exit();
 }
 $saved = " ";
-if (isset($_POST['submitted'])) {
-    $studentMark = $_POST['student']." ".$_POST['subject']." ".$_POST['mark']."\n";
+if (isset($_POST['student']) and isset($_POST['subject']) and isset($_POST['mark'])) {
+    $studentMark = $_POST['student'].",".$_POST['subject'].",".$_POST['mark']."\n";
+    $studentMark = str_replace("\r\n", "", $studentMark);
     file_put_contents($marksFilename, $studentMark, FILE_APPEND);
     $saved = "Išsaugota";
 }
-$studentOptions = "";
+$studentOptions = " ";
 @$peopleFile = fopen(@$peopleFilename, "r");
-$peopleFilename = 'people.txt';
-for ($line = fgets($peopleFile); !feof($peopleFile); $line = fgets($peopleFile)) {
-    $studentOptions .= "<option>{$line}</option>";
+for ($line = fgets($peopleFile); !feof($peopleFile); $line = fgets($peopleFile)) { 
```

Comment:
https://stackoverflow.com/questions/6379947/in-php-when-using-fgets-to-read-from-file-how-to-exclude-brake-row